readme-vars.yml

```diff
@@ -0,0 +1,50 @@
+---
+
+# project information
+project_name: overseerr
+project_url: "https://overseerr.dev/"
+project_logo: "https://raw.githubusercontent.com/linuxserver/docker-templates/master/linuxserver.io/img/overseerr.png"
+project_blurb: |
+  [{{ project_name|capitalize }}]({{ project_url }}) is a free and open source software application for managing requests for your media library.
+
+project_lsio_github_repo_url: "https://github.com/linuxserver/docker-{{ project_name }}"
+
+# supported architectures
+available_architectures:
+  - { arch: "{{ arch_x86_64 }}", tag: "amd64-latest"}
+  - { arch: "{{ arch_arm64 }}", tag: "arm64v8-latest"}
+  - { arch: "{{ arch_armhf }}", tag: "arm32v7-latest"}
+
+# development version
+development_versions: false
+development_versions_items:
+  - { tag: "latest", desc: "Stable releases from GitHub" }
+  - { tag: "develop", desc: "Development releases from commits in upstream develop branch" }
+
+# container parameters
+common_param_env_vars_enabled: true
+param_container_name: "{{ project_name }}"
+
+param_usage_include_env: true
+param_env_vars:
+  - { env_var: "TZ", env_value: "America/New_York", desc: "Specify a timezone to use EG America/New_York"}
+
+param_usage_include_ports: true
+param_ports:
+  - { external_port: "5055", internal_port: "5000", port_desc: "Port for Overseerr's web interface." }
```

Comment:
shouldn't the internal be 5055?

Reply:
Resolved in abb8948eb611dfc1654218f6cbd6b3691f53ff91